### PR TITLE
Traitor, a traitor is a traitor.

### PR DESCRIPTION
**Pull request by optimumtact**
Means that one of the crew members works for a criminal Syndicate and
must something to seize / kill someone / someone to protect and dump
from that Durkee on the shuttle. At English stations the word everyday,
for Russians for the mention of the word "traitor" is not in the chat
(IC) admins are stroking the head with a bangammer. So this word and
memetic.

### CODE REVIEW

**Comment by RemieRichards** [comment]
Uh... sure... why not.

**Comment by optimumtact** [comment]
https://translate.google.com/translate?sl=auto&tl=en&js=y&prev=_t&hl=en&ie=UTF-8&u=%2F%2Flurkmore.to%2FSS13&edit-text=&act=url

**Comment by RemieRichards** [comment]
>multiplayer rolevka

So THAT'S what we've been missing! _rolevka!_

**Comment by Artemiuz28** [comment]
"bangammer"... Is this intentional?
Also, don't take lurkmore seriously. It's like 1d4chan multiplied by 2.

**Comment by optimumtact** [comment]
No I assume it was translated badly, but that's what makes it funny

**Comment by RemieRichards** [comment]
I hope we never solve perfect automated translation, we'd lose wonderful times like these

**Comment by FantasticFwoosh** [comment]
The grammar was gammy from the ankle up.

File: code/game/objects/items/weaponry.dm [hunk 1]
```diff
@@ -18,8 +18,11 @@
 		return (BRUTELOSS|FIRELOSS|TOXLOSS|OXYLOSS)
 
 /obj/item/banhammer/attack(mob/M, mob/user)
-	to_chat(M, "<font color='red'><b> You have been banned FOR NO REISIN by [user]<b></font>")
-	to_chat(user, "<font color='red'>You have <b>BANNED</b> [M]</font>")
+	if(user.zone_selected == "head")
+		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");
```
Comment (MrPerson) [inline code/game/objects/items/weaponry.dm]: bangammer?

**Comment by ExcessiveUseOfCobblestone** [comment]
TERRORISM IN THE ATMOSPHERE

**Review by Cyberboss — APPROVED** [review]

**Comment by Artemiuz28** [comment]
I know russian, and Lurkmore means what it means. Sure, bad translation makes a part of weirdness, but it's really minor. Lurkmore intentionally uses an overwhelming amount of slang, very exaggerated contents and references to Russian memes.

**Pull request merged**